### DISCUSSION TITLE
v0.14.2

### DISCUSSION THREAD

**Discussion by ledermann** [solectrus/solectrus#2710] [Updates]
### Bugfixes

- Fixed missing visualization on NOW tab when MQTT is used and some data is `NULL` instead of `0` (#2647)
- Fixed high memory usage, caused by starting multiple Puma instances by default (#2625)
- Minor fixes

### Smaller improvements

- CSS animation added again, but with lower energy consumption and respecting [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) (#2595)
- At night, the homepage renders `house_power` chart instead of `inverter_power` by default
- Removed SEO stuff, added `robots.txt` and meta tag "robots" to disallow all bots (#2687)
- Added throttling (max. 300 requests per 5 minutes) to lock out spammy clients, useful for public installations
- Docker: Container can be started with custom command, Puma is started by default
- Updated dependencies

<hr /><em>This discussion was created from the release <a href='https://github.com/solectrus/solectrus/releases/tag/v0.14.2'>v0.14.2</a>.</em>

**Comment by OnkelMF** [comment]
Hi Georg,

habe gestern morgen das Update (wie gehabt nach Deiner Anleitung) auf die 14.2 gemacht - soweit so gut, lief auch alles zunächst.
Plötzlich nachmittags zeigte Solectrus "Keine Verbindung" oben rechts. Also Log aktualisiert weil ich dachte der übliche NPU-Fehler - im Log war aber nichts zu sehen, also runter in den Keller - Verbindung am Speicher steht. Also nochmal sudo reboot und docker ps probiert - scheint alles zu laufen. Aber ich kann weder über die lokale IP noch über meine dynDNS auf meine Daten zugreifen (solectrus öffnet sich nicht mehr). Ich habe aber nichts am Router verändert, am Raspberry seit dem Update auf 14.2 auch nicht mehr. Die lokale IP kann ich anpingen im Heimnetzwerk - was kann ich tun ? Mein Senec und die Senec-App laufen also der V3 liefert Daten (log geht auch). Also stimmt irgendwas am Raspberry nicht !? Kann ich noch mehr testen um den Fehler einzugrenzen ? Ich verzweifel ohne mein Solectrus unterwegs :(

Grüße aus dem Süden Berlins
